recipesearch/src/Recipes/SearchBar/reducer.js

```diff
@@ -0,0 +1,15 @@
+const initialState = {
+  searchResults: []
```

Comment:
It's better to set `searchResults`' initial value to `null` because an empty array is truthy

Reply:
Alright @edmondatto .
Let me make the relevant changes.

Reply:
I have made the changes